### ISSUE TITLE
Redirect to tutorial ("/" to "/docs")

### ISSUE THREAD

**Issue by clairefro** [redwoodjs/learn.redwoodjs.com#5]
~Make the homepage more "Redwoody"~

Accessing "/" defaults to a homepage splash, mostly still the docusuarus defaults. 

We don't really need a homepage for now as we just have tutorial content. Let's set a up a soft redirect from "/" to "/docs"

![image](https://user-images.githubusercontent.com/9841162/103693253-da641400-4f4d-11eb-880a-e2cd49abf72b.png)

**Comment by cannikin** [comment]
Could always follow the latest design trend and just Make It Look Like Stripe.com!

**Comment by cannikin** [comment]
BTW the reds I ended up using at the default Tailwind "breakpoints":

```
'100': '#FDF8F6',
'200': '#FAEAE5',
'300': '#F3C7BA',
'400': '#EBA48E',
'500': '#E38163',
'600': '#DC5E38',
'700': '#BF4722',  <-- actual brand color for the logo
'800': '#682712',
'900': '#341309',
```

![image](https://user-images.githubusercontent.com/300/103702433-84966880-4f5b-11eb-88cb-c60893313a34.png)

**Comment by cannikin** [comment]
If the only thing it has right now is the tutorial, maybe it should just jump to the Welcome page? And then when we start moving more docs there we'll come up with a homepage that has some road signs to get you where you want to go.

This way it becomes future-us problems and now-us get to relax. I'm sure we won't regret that. 😬

**Comment by clairefro** [comment]
Sounds good to me, gonna redact the issue

**Comment by mbush92** [comment]
I opened PR #11  that uses the Docusaurus `Redirect` component from Docusaurus router to redirect the user from the home page to the `/docs` page.  Since the discussion was that the home page would be worried about in the future I went ahead and removed any content that was not needed from the home page component.

**Comment by mbush92** [comment]
@clairefro Can we close this issue now?

**Comment by clairefro** [comment]
Resolved by #11
